SchedulerJobs/SchedulerJobs.Services/AnonymiseHearingsConferencesDataService.cs

```diff
@@ -34,12 +39,27 @@ private async Task DeleteUserAsync()
             // get users that do not have hearings in the future and have had hearing more than 3 months in the past. 
             // (exclude judges, vhos, test users, performance test users.
             var usersToDelete = await _bookingsApiService.GetUsersWithClosedConferencesAsync();
-            if (usersToDelete != null && usersToDelete.Usernames != null)
+            
+            _logger.LogInformation($"AnonymiseHearingsConferencesDataService: Found {usersToDelete.Usernames.Count} users to delete");
```

Comment:
Please don't use interpolated strings in the logger.

should be written as 

` _logger.LogInformation("AnonymiseHearingsConferencesDataService: Found {UsersToDeleteCount} users to delete", usersToDelete.Usernames.Count);`